### PR TITLE
Example for returning to REACT_APP_DASHBOARD_URL

### PR DESCRIPTION
**Pull request by achen2401**
per [convo](https://cirg.slack.com/archives/C03HTRD8RRS/p1663622068982799?thread_ts=1663622061.399429&cid=C03HTRD8RRS) in Slack

Add example for returning to pre-defined (f)EMR URL

### CODE REVIEW

**Review by pbugni — APPROVED** [review]
Perfect - thank you!

File: src/components/Summary.js [hunk 1]
```diff
@@ -51,6 +54,7 @@ export default function Summary() {
   const contextContent = React.useContext(FhirClientContext);
   const [patient, setPatient] = React.useState(null);
   const [error, setError] = React.useState(null);
+  const returnURL = getEnv("REACT_APP_DASHBOARD_URL");
```
Comment (hannahburkhardt) [inline src/components/Summary.js:57]: Should this be something like `getEnv("REACT_APP_DASHBOARD_URL") + "/clear_session";`?

File: src/components/Summary.js [hunk 2]
```diff
@@ -121,13 +125,15 @@ export default function Summary() {
...
-        <div>
+        <Stack spacing={2}>
           <h2>Hello World SoF App</h2>
           <div>
             Name: {patient.name[0].given.join(" ") + patient.name[0].family}
           </div>
           <div>DOB: {patient.birthDate}</div>
-        </div>
+          {/* example of an UI button, on clicking of which will go to the (f)EMR URL defined by an environment variable */}
+          {returnURL && <div><Button variant="contained" href={returnURL}>Return to Patient List</Button></div>}
```
Comment (hannahburkhardt) [inline src/components/Summary.js:135]: Does this need to be enclosed in a div tag?

File: src/components/Summary.js (hunk 1 above)
Comment (achen2401) [reply src/components/Summary.js:57]: thank you, fixed.

File: src/components/Summary.js (hunk 2 above)
Comment (achen2401) [reply src/components/Summary.js]: thank you, fixed

**Pull request merged**